### DISCUSSION TITLE
Playback Exception on Windows 10

### DISCUSSION THREAD

**Discussion by RRRitchey** [OpenSageTV/Community#127] [General]
I just installed SageTV 7.1.4.676 on Windows 10.  The video sources setup just fine.  When I tried to view some live TV I got the following error: 
"Sage.Playback Exception: ERROR (-4,0x80040217) There was a problem rendering the video portion of the content for playback"

I have the same version of SageTV running just fine on 3 Windows 7 computers.  Installation went off without a hitch on Windows 10. Not sure what could be wrong here.  Searched but found no other discussions on this. 

Thanks

**Comment by trk2** [comment]
I would at Setup->Detailed Setup->Video / Audio and make sure the video renderer is set to EVR and the decoders are set to LAV Video or LAV Audio.

**Reply by RRRitchey** [reply]
Hi, thanks for the suggestions.  The video renderer is set to EVR.  As for the decoders, I could not find LAV Video anywhere.  I told it to list all known decoders and LAV was not in the list.  Should these be available or do I need to add them? 
For a bit more information on the system: Intel i7-7700, 32GB RAM, 64-bit Windows 10 Enterprise 22H2.  
Thanks

**Reply by trk2** [reply]
Sorry, I didn't pay attention when you said you were installing SageTV 7.  Is there any reason why you are not running SageTV 9?   LAV Filiters is included with the Sage installer in later versions.   For your version you have to download [lav filters](https://github.com/Nevcairiel/LAVFilters/releases) and install them separately.

**Reply by RRRitchey** [reply]
Sorry, I mistyped.  It is version 9.1.4.676.  So The LAV filters should have been there?  I will try downloading the versions you linked. Thanks,

**Reply by RRRitchey** [reply]
Thanks, that did it.  Don't know why those were not there since I was installing V9. It always worked without a hitch on Windows 7.

**Reply by wayner9** [reply]
If I remember correctly there were licensing $ reasons why some decoders that were included in Win7 that were not included in later versions.

**Reply by RRRitchey** [reply]
Thanks for the explanation.  Good to know.
